docs/guide/introduction.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+`asdf` is a tool version manager. All tool version definitions are contained within one file (`.tool-versions`) which you can check-in to your project's Git repository to share with your team, ensuring everyone is using the **exact** same versions of tools.
```

Comment:
I like the fact that you've mentioned the `.tool-versions` file in the first paragraph. It is just as much an API as the CLI commands are, but it isn't prominently documented on our existing site.

Reply:
Great, hopefully with this here and the guide people will find it more clear it needs to be set for `asdf` to work.